misc/scripts/deploy-tools.sh

```diff
@@ -89,6 +94,49 @@ function deployplugin {
     done
 }
 
+function deploypluginwindows {
+    log "Compressing source into $PLUGIN_SRC_ZIP..."
+    cd $PLUGIN_SRC_DIR
+    rm -f $PLUGIN_SRC_ZIP
+    stashName=`git stash create`;
+    git archive --format zip --output $PLUGIN_SRC_ZIP $stashName
+
+    for ip in $IP_LIST
+    do
+        TARGET=root@$ip
+
+        log "Cleaning up older files from $TARGET..."
+        $SSH $TARGET <<-EOF
+            powershell Remove-Item -Recurse -Force $WIN_PLUGIN_SRC_DIR
+		EOF
+        # EOF needs to be tab indented
+
+        log "Transferring $PLUGIN_SRC_ZIP to $TARGET..."
+        scp $PLUGIN_SRC_ZIP $TARGET:$WIN_TEMP_DIR
+
+        log "Extracting $PLUGIN_SRC_ZIP on $TARGET..."
+        $SSH $TARGET "powershell Expand-Archive $WIN_TEMP_DIR\\$PLUGIN_SRC_ZIP -DestinationPath $WIN_PLUGIN_SRC_DIR"
+
+        log "Building windows plugin on $TARGET..."
+        $SSH $TARGET <<-EOF
```

Comment:
can you stay with single line ? `$SSH $TARGET "cd $WIN_PLUGIN_SRC_DIR; build.bat"`